examples/Atlas/runAtlasBalancingSplitPerturb.m

```diff
@@ -13,9 +12,7 @@ function runAtlasBalancingSplitPerturbation(example_options)
 checkDependency('lcmgl');
 
 if nargin<1, example_options=struct(); end
-example_options = applyDefaults(example_options, struct('use_mex', true,...
-                                                        'use_bullet', false,...
-                                                        'quiet', true,...
+example_options = applyDefaults(example_options, struct('use_bullet', false,...
                                                         'perturb_body', 'mtorso',...
```

Comment:
nevermind.  i guess since you also removed it from the documentation for the method above, it must have been intentional.